modules/ROOT/pages/analytics-event-api.adoc

```diff
@@ -8,22 +8,19 @@ endif::[]
 
 API Manager gathers analytics data for each API that you manage.  You can generate this data in reports and charts, using the Analytics Event API. You can create reports for a specific environment or for multiple APIs, and you can create different reports for the same API.
 
-If your organization's API use exceeds 20,000 requests per 10 minutes, using this API causes throttling. Consider the alternative of configuring the API gateway to send events directly to monitoring systems.
+== Requirements and Restrictions
 
+The Analytics Events API provides a one month retention policy and a data availability delay of up to 10 minutes. It also enforces a usage limit of 10 requests per minute, with a maximum of 20,000 events per request. Request rates higher than 10 requests per minute are throttled.
 
-== Requirements and Restrictions
+The API is designed to create ad-hoc reports. It is not intended as a data export tool. 
 
-The Event API is designed to create ad-hoc reports. It is not intended as a data export tool. Before you start using the API to create reports, consider the following requirements and restrictions:
+Before you start using the API to create reports, consider the following requirements and restrictions:
 
 * Do not allow calls to overlap time intervals.
-* Retrieve all the data that you need in a single request, but note that data retrieval is limited to 20,000 events.
 * Do not repeat the same request for every API that you have.
 //Why?
-* Although the Analytics Event API enforces a limit of 10 requests per minute, best practice is 1 request per minute.
-+
-Request rates higher than 10 request per minutes are throttled.
-* The analytics data for events is retained for one month, with a data availability delay of up to 10 minutes.  
-* Do not use the Analytics Event API for data synchronization. 
+* Although the Analytics Events API enforces a limit of 10 requests per minute, best practice is 1 request per minute. 
```

Comment:
", best practice" -> ", the best practice" or ", a best practice" ?

Reply:
Good catch! Fixed.